addons/website/static/src/systray_items/relative_publish_time.js

```diff
@@ -0,0 +1,32 @@
+import { RelativeTime } from "@mail/core/common/relative_time";
+import { _t } from "@web/core/l10n/translation";
+
+const MINUTE = 60 * 1000;
+
+export class RelativePublishTime extends RelativeTime {
+    static props = {
+        datetime: Object,
+        negativeDeltaCallback: {
+            type: Function,
+            optional: true,
+        },
+    };
+    /**
+     * @override
+     */
+    computeDeltaAndRelativeTime() {
+        const delta = this.props.datetime.ts - Date.now();
+        let relativeTime;
+        if (delta < 0) {
+            this.props.negativeDeltaCallback();
+            clearTimeout(this.timeout);
+        } else if (delta < MINUTE) {
+            relativeTime = _t("Published shortly");
+        } else {
+            relativeTime = _t("Published %(datetime)s", {
+                datetime: this.props.datetime.toRelative(),
+            });
+        }
+        return [delta, relativeTime];
+    }
+}
```

Comment:
This looks like more code than if you just duplicated our component and adapted it to your needs.

And it's a lot harder for us to maintain it with this override in the way, so I'd say just duplicate it.

Reply:
@seb-odoo You're probably right. I don't want to make your life harder :) Thx for the comment.